runtime/runtime/src/prefetch.rs

```diff
@@ -85,7 +89,10 @@ impl TriePrefetcher {
     /// Start prefetching data for processing the receipts.
```

Comment:
Not part of the PR but this might be worth some drive-by cleanup.

```suggestion
    /// Starts prefetching data for processing the receipts.
```